modules/fleet-app-operator-permissions/main.tf

```
@@ -77,7 +79,7 @@ resource "google_gke_hub_scope_rbac_role_binding" "scope_rbac_user_role_bindings
   scope_id                   = var.scope_id
   user                       = each.key
   role {
-    predefined_role = var.role
+    var.custom_role != "" ? custom_role = var.custom_role : predefined_role = var.role
```

You will probably need something similar to this (untested) to set both the `custom_role` and `predefined_role` fields:

```suggestion
    custom_role = (var.custom_role != "" ? var.custom_role : null)
    predefined_role = (var.custom_role != "" ? null : var.role)
```